_docs-sources/foundations/landing-zone/prerequisites.md

```diff
@@ -16,45 +18,78 @@ To use AWS Control Tower you must first enable it, which requires that you have
   This account will become the root of your multi-account setup after enabling Control Tower.
   :::
 
-1. Three(3) new unique email addresses for your logs, shared, and security (audit) accounts.
+1. Three(3) new unique email addresses for your logs, shared, and security (audit) accounts. It's important to note that these email addresses cannot be already associated with an AWS root login.
+
 1. A KMS Key for encrypting Control Tower Resources with a suitable permissions policy (See Guide Below)
+
 1. A home region selection where your Control Tower configuration will reside.
    :::caution
    Your home Region is the AWS Region in which you'll run most of your workloads or store most of your data. It **cannot** be changed after you've set up your AWS Control Tower landing zone. For more information about how to choose a home Region, see [Administrative tips for landing zone setup](https://docs.aws.amazon.com/controltower/latest/userguide/tips-for-admin-setup.html).
    :::
 
-### KMS Key Creation
+---
```

Comment:
This rule confuses me again. Perhaps we're interpreting the structure of the page differently. Then again, if it's wrong, maybe we should address the hierarchy of the content rather than add in rules to divide it arbitrarily. I do think the hierarchy of this page is a little confusing in its own right. Collapsing the images seems like a win, but I'd leave the rules out and we can discuss a better way to structure the content.

For example, there's a whole section for pipelines, which just has one sentence which points to the Pipelines docs. That's actually misleading, because if you follow all the docs to set up a standalone Pipeline you'll be worse off, since that won't support Landing Zone, which requires a multi-account configuration. We only tell people to install Pipelines down in the Initial Setup page of the IaC Foundations section. But that requires having Control Tower already configured. Eek!

This really does need some larger rework. Perhaps we should just remove the Pipelines section from the prerequisites here for now, lest we lead people astray. The Next Steps section doesn't add much either, to be honest.

Reply:
Yes I agree larger rework is needed, there are definitely some gaps. 

To address your initial comment on the rule usage, it's similar to the other comment I left in that I was just trying to break the page up for readability. It may not be the right choice here. (and very well may be better addressed by reworking the organization of the guides instead).

Reply:
Okay, here's a concrete proposal I feel we could do with this change. Let me know your thoughts:
* Remove the "Gruntwork Pipelines" section
* Remove the "Next steps" section
* Move the "Creating the KMS key" section into a details block within item 3 in the list of pre-reqs
* Remove the "Prerequisites for enabling AWS Control Tower" _header_ (it's redundant, since at this point, the only content on the page is the numbered list of 4 pre-reqs)
* Remove all the rules, which should no longer be needed on the much-shortened page

Reply:
Great suggestions, this is much more simplified. The only suggestions I wasn't able to incorporate was embedding the entire Creating a KMS key sections into a details block. For whatever reason I was unable to get details blocks to render inside another details block (the screenshots). I think it looks better though, so let me know what you think about how it looks now! I can keep tinkering with it if it's still not quite right.